packages/gatsby-source-try-ghost/gatsby-node.js

```diff
@@ -146,8 +146,8 @@ const prefetchPostAndPageNodes = (existingNodes, sourceNodeFields, settings, fet
 
     const removeOrUpdatePosts = api.posts.browse(prefetchOptions).then(async (posts) => {
         log(`Prefetched Posts: ${posts.length}`)
-        if (fetchPostProcessors && fetchPostProcessors.post) {
-            posts = posts.map(post => fetchPostProcessors.post(post)).filter(post => !!post)
+        if (filter && filter.posts) {
```

Comment:
1. Also check that filter.posts is indeed a function, e.g. typeof filer.posts === 'function'
2. Maybe remove curly brackets and do it as a one-liner (also in other occurrences)